api/paidAction/transfer.js

```diff
@@ -0,0 +1,57 @@
+import { satsToMsats } from '@/lib/format'
+import { notifyDeposit } from '@/lib/webPush'
+import { PAID_ACTION_PAYMENT_METHODS } from '@/lib/constants'
+
+export const anonable = true
+
+export const paymentMethods = [
+  PAID_ACTION_PAYMENT_METHODS.P2P
+]
+
+export async function getCost ({ sats }) {
+  return satsToMsats(sats)
+}
+
+export async function invoiceablePeer ({ targetUserId }, { models }) {
+  return targetUserId
+}
+
+export async function perform ({ invoiceId, sats, description, descriptionHash, comment, targetUserId }, { me, tx }) {
+  const invoice = await tx.invoice.update({
+    where: { id: invoiceId },
+    data: {
+      comment
+    }
+  })
+  await notifyDeposit(targetUserId, invoice)
```

Comment:
Won't this notify them every time a deposit is *attempted* not every time a deposit *succeeds*?